golem_messages/message.py

```diff
@@ -1137,6 +1138,44 @@ def __init__(self, subtask_id=None, **kwargs):
         super(MessageSubtaskPaymentRequest, self).__init__(**kwargs)
 
 
+class MessageAckReportComputedTask(Message):
+    TYPE = TASK_MSG_BASE + 29
+
+    __slots__ = [
+        'subtask_id',
+    ] + Message.__slots__
+
+    def __init__(self, subtask_id=None, **kwargs):
+        self.subtask_id = subtask_id
+        super().__init__(**kwargs)
+
+
+@unique
+class MessageRejectReportComputedTaskReason(Enum):
+    TASK_TIME_LIMIT_EXCEEDED = 1
+    SUBTASK_TIME_LIMIT_EXCEEDED = 2
+    GOT_MESSAGE_CANNOT_COMPUTE_TASK = 3
+    GOT_MESSAGE_TASK_FAILURE = 4

```

Comment:
Since this enum is tightly coupled with `MessageRejectReportComputedTask` could you move it inside that class and name it `REASON`? Also according to discussion in #10 I will appreciate using more sophisticated labels than 1,2,3,4 :)

Reply:
done